src/openfermion/ops/_binary_operator_test.py

```diff
@@ -0,0 +1,108 @@
+import unittest
+from _binary_operator import SymbolicBinary
+
+
+class SymbolicBinaryTest(unittest.TestCase):
```

Comment:
These are nice tests but unfortunately you will need a few more. This is the downside to introducing a large class like this with all the magic methods. Coveralls will show you which lines are covered and which aren't. Apparently you are covering 156 relevant lines but missed 43.